MediaBrowser.Api/System/ActivityLogService.cs

```diff
@@ -53,7 +53,7 @@ public object Get(GetActivityLogs request)
                 (DateTime?)null :
                 DateTime.Parse(request.MinDate, null, DateTimeStyles.RoundtripKind).ToUniversalTime();
 
-            var result = _activityManager.GetActivityLogEntries(minDate, request.HasUserId, request.StartIndex, request.Limit);
+            var result = _activityManager.GetPagedResult(request.StartIndex, request.Limit);
```

Comment:
why minDate is not used?

Reply:
🤷 Looks like that's been there for 6 years